### PR TITLE
Add template tag support

### PR DESCRIPTION
**Pull request by azhiv**
Also, add an empty implemention for blockName because otherwise template tag doesn't have anything to import into the tag.

### CODE REVIEW

File: test-app/.eslintrc.js [hunk 1]
```diff
@@ -24,6 +24,42 @@ module.exports = {
   },
   rules: {},
   overrides: [
+    {
+      files: ['**/*.{js,ts}'],
+      plugins: ['ember'],
+      parser: '@typescript-eslint/parser',
```
Comment (andreyfel) [inline test-app/.eslintrc.js:30]: Why do you add '`typescript-eslint` if there is no TS?

File: test-app/.eslintrc.js [hunk 2]
```diff
@@ -24,6 +24,42 @@ module.exports = {
...
+      extends: ['eslint:recommended', 'plugin:ember/recommended'],
+      rules: {
+        'ember/no-replace-test-comments': 'error',
+      },
+    },
+    {
+      files: ['**/*.gts'],
```
Comment (andreyfel) [inline test-app/.eslintrc.js:37]: No gts files in the project.

File: test-app/package.json [hunk 3]
```diff
@@ -58,6 +58,7 @@
     "ember-resolver": "11.0.1",
     "ember-source": "5.5.0",
     "ember-source-channel-url": "3.0.0",
+    "ember-template-imports": "^4.0.0",
```
Comment (andreyfel) [inline test-app/package.json:61]: Pin dependency

File: test-app/package.json [hunk 4]
```diff
@@ -68,6 +69,7 @@
     "eslint-plugin-qunit": "8.0.1",
     "loader.js": "4.7.0",
     "prettier": "3.2.4",
+    "prettier-plugin-ember-template-tag": "^2.0.0",
```
Comment (andreyfel) [inline test-app/package.json:72]: Pin dependency

**Review by andreyfel — CHANGES_REQUESTED** [review]

File: test-app/.eslintrc.js [hunk 5]
```diff
@@ -24,6 +24,21 @@ module.exports = {
...
+      plugins: ['ember'],
+      extends: ['eslint:recommended', 'plugin:ember/recommended'],
+    },
+    {
+      files: ['**/*.gjs'],
+      parser: 'ember-eslint-parser',
+      plugins: ['ember'],
+      extends: [
+        'eslint:recommended',
+        'plugin:ember/recommended',
```
Comment (andreyfel) [inline test-app/.eslintrc.js:38]: Do you need to specify these extends in the override if they are specified in the base config?

**Review by andreyfel — CHANGES_REQUESTED** [review]

File: test-app/.eslintrc.js [hunk 6]
```diff
@@ -24,6 +24,21 @@ module.exports = {
   },
   rules: {},
   overrides: [
+    {
+      files: ['**/*.{js}'],
```
[the remark this answers is not in the document]
Comment (azhiv) [reply test-app/.eslintrc.js]: Done

File: test-app/.eslintrc.js (hunk 5 above)
Comment (azhiv) [reply test-app/.eslintrc.js]: Done

File: test-app/package.json (hunk 4 above)
Comment (azhiv) [reply test-app/package.json]: Done

File: test-app/.eslintrc.js (hunk 2 above)
Comment (azhiv) [reply test-app/.eslintrc.js]: Done

File: test-app/.eslintrc.js (hunk 1 above)
Comment (azhiv) [reply test-app/.eslintrc.js]: Done

**Review by andreyfel — APPROVED** [review]

**Pull request merged**